README.md

```diff
@@ -22,50 +22,52 @@ The attributes listed below are used in *components.json* to configure **Tabs**,
 
 **_parentId** (string): An identifier that links the component to the parent block.
 
-**_type** (string): The type of the particular item. Examples include block and component.	
+**_type** (string): The type of the particular item. Examples include block and component.
 
 **_component** (string): This value must be: `tabs`.
 
 **_classes** (string): CSS class name to be applied to the container div. The class must be predefined in one of the Less files. Separate multiple classes with a space.
 
 **_layout** (string): This defines the horizontal position of the component in the block. Acceptable values are `full`, `left` or `right`.
 
-**title** (string): The title of the particular item.	
+**title** (string): The title of the particular item.
 
-**displayTitle** (string): This is the title that Adapt displays when viewing a course.	
+**displayTitle** (string): This is the title that Adapt displays when viewing a course.
 
-**body** (string): The body text content of the particular item.	
+**body** (string): The body text content of the particular item.
 
 **instruction** (string): This optional text appears above the component. It is frequently used to
-guide the learner’s interaction with the component.  
+guide the learner’s interaction with the component.
 
 **_tabLayout** (string): Defines the layout of the tabs. Acceptable values are `left` and `top`. By default, component displays `left` layout for medium screen size and below.
 
-**_items** (array): Multiple items may be created. Each *_item* represents one element of the tabs component and contains values for **tabTitle**, **title**, **body**, and **_graphic**. 
+**_items** (array): Multiple items may be created. Each *_item* represents one element of the tabs component and contains values for **tabTitle**, **title**, **body**, and **_graphic**.
 
 >**tabTitle** (string): This text is displayed in the actual tab heading. Recommend keeping this title short.
 
 >**title** (string): This text is displayed as the content panel's header. It is displayed when the tab has been selected.
 
->**body** (string): This content will be displayed when the learner opens this content panel. It may contain HTML.  
+>**body** (string): This content will be displayed when the learner opens this content panel. It may contain HTML.
 
 >**_graphic** (object): An optional image which is displayed below the item body when the learner opens this content panel. It contains values for *src* and *alt*.
```

Comment:
Change last sentence to "It contains valus for src, alt and _classes"

Reply:
The classes are applied to the item level rather than the graphic level

Reply:
Makes sense, I'll move it.

Reply:
Done